plugins/outputs/zabbix/README.md

```diff
@@ -0,0 +1,406 @@
+# Zabbix Output Plugin
+
+This plugin send metrics to [Zabbix](https://www.zabbix.com/) via
+[traps][traps].
+
+It has been tested with versions
+[3.0](https://www.zabbix.com/documentation/3.0/en/manual/appendix/items/trapper)
+,
+[4.0](https://www.zabbix.com/documentation/4.0/en/manual/appendix/items/trapper)
+and
+[6.0](https://www.zabbix.com/documentation/6.0/en/manual/appendix/items/trapper)
+.
+
+[traps]: https://www.zabbix.com/documentation/current/en/manual/appendix/items/trapper
+
+It should work with newer versions as long as Zabbix does not change the
+protocol.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Send metrics to Zabbix
+[[outputs.zabbix]]
+  ## Address and (optional) port of the Zabbix server
+  address = "zabbix.example.com:10051"
+
+  ## Send metrics as type "Zabbix agent (active)"
+  # agent_active = false
+
+  ## Add prefix to all keys sent to Zabbix.
+  # prefix = "telegraf."
+
+  ## Name of the tag that contains the host name
+  # host_tag = "host"
+
+  ## Skip measurement prefix to all keys sent to Zabbix.
+  # skip_measurement_prefix = false
```

Comment:
Is this related to the prefix option above? If so, we should remove `skip_measurement_prefix` and if `prefix` is set to an empty string, skip the prefix.

Reply:
No, that `skip_measurement_prefix` is the measurement name.
For example, for a metric like: `mem,host=foo free=123`, having `prefix=""`, the zabbix key for each case would be:

With `skip_measurement_prefix=false`: `mem.free`

With `skip_measurement_prefix=true`: `free`

Reply:
Does this mean if a field is used across different metrics there is a collision? This feels like a footgun.

Reply:
Correct. It not a parameter that should be normally used, but give more flexibility. We are currently using it to forward metrics between two systems.